### DISCUSSION TITLE
When will the binding of vertex related functions be released?

### DISCUSSION THREAD

**Discussion by whu-giser** [uber/h3-py#391] [Q&A]
11

**Comment by ajfriend** [comment]
Awesome to see folks are interested in having these available! I'm hoping to release a new beta with #388 in the next few days.

Also, while I did want to separate the feature in #338 from a release, I'd be happy if someone wanted to put together a pure release PR before I get to it. :)

cc: @Shiran-Yuan

**Reply by ajfriend** [reply]
Should now be released to PyPI as v4.0.0b6. Let me know if you have any issues.
